### PR TITLE
[DEST-757][CC-4858] Add product list viewed event to Monetate

### PR DESCRIPTION
**Pull request by CarlosMecha**
- Original PR: #137 
- [DEST-757](https://segment.atlassian.net/browse/DEST-757)
- [CC-4854](https://segment.atlassian.net/browse/CC-4854)

### CODE REVIEW

File: integrations/monetate/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@segment/analytics.js-integration-monetate",
   "description": "The Monetate analytics.js integration.",
-  "version": "2.2.0",
```
Comment (ccnixon) [inline integrations/monetate/package.json]: If it's not a breaking change, should we just bump by a minor version?

Comment (CarlosMecha) [reply integrations/monetate/package.json]: Sadly is a breaking change. It changes how `product viewed` behaves :(

Comment (ccnixon) [reply integrations/monetate/package.json]: Oh it doesn't just add a new handler for `productListViewed`?

File: integrations/monetate/lib/index.js [hunk 2]
```diff
@@ -40,7 +40,8 @@ Monetate.prototype.initialize = function() {
   if (this.options.retail) {
     this.options.events = {
       orderCompleted: 'addPurchaseRows',
-      productViewed: 'addProducts',
+      productViewed: 'addProductDetails',
```
Comment (ccnixon) [inline integrations/monetate/lib/index.js]: Is this the breaking change?

Comment (CarlosMecha) [reply integrations/monetate/lib/index.js]: Yep!

**Pull request merged**